src/pages/assessments/index.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { GetStaticProps } from "next";
+import Link from "next/link";
+import Head from "next/head";
+
+import { PageBanner } from "../../components/Banner";
+import { DemoCourseAction } from "../../components/DemoCourse";
+import Stack from "../../components/Stack";
+
+import { getAssessments } from "../../lib/gallery/assessments";
+
+interface Assessment {
+  title: string;
+  slug: string;
+  summary: string;
+}
+
+interface AssessmentIndexProps {
+  assessments: Assessment[];
+}
+
+const AssessmentIndex: React.FC<AssessmentIndexProps> = ({ assessments }) => {
+  return (
+    <React.Fragment>
+      <Head>
+        <title>Assessments | PrairieLearn</title>
+      </Head>
+      <PageBanner
+        title="Assessments"
+        text="Building different types of activities for your class"
+      />
+
+      <div className="container my-5">
+        <div className="mb-3">
+          <Stack spacing={3}>
+            {assessments.map((assessment) => {
+              const assessmentHref = `/gallery/assessment/${assessment.slug}`;
+              return (
+                <article key={assessment.slug}>
+                  <h3 className="h5">
+                    <Link href={assessmentHref}>
+                      <a>{assessment.title}</a>
+                    </Link>
+                  </h3>
+                  <p className="mb-0">{assessment.summary}</p>
+                </article>
+              );
+            })}
+          </Stack>
+        </div>
+      </div>
+
+      <DemoCourseAction
+        title="Try our demo course!"
+        text="Explore the demo course to see how this all come together"
+        button="Demo course"
```

Comment:
I think we should try to make these buttons a stronger call to action - maybe "View demo course" or "Explore demo course"?

Reply:
changed to "View demo course" for now.